AutocompleteClientTests/FW/Logic/Worker/ConstructorIOIntegrationTests.swift

```diff
@@ -222,11 +223,11 @@ class ConstructorIOIntegrationTests: XCTestCase {
         self.constructor.recommendations(forQuery: query, completionHandler: { response in
             let cioError = response.error as? CIOError
             XCTAssertNil(cioError)
-
+            
```

Comment:
Looks like there are still some random white spaces showing up in this file. Do they not show up in the lint errors for you?